.eslintrc

```diff
@@ -1,4 +1,5 @@
 {
+  "parser": "babel-eslint",
```

Comment:
Do you think that putting all of the eslint needed changes in a separate commit would be possible?